crates/host/src/wasmbus/mod.rs

```diff
@@ -3588,6 +3518,7 @@ impl Host {
             .trim_start_matches('.')
             .split('.')
             .skip(1);
+        info!("handling ctl request on {}", subject);
```

Comment:
This should probably be a `trace`. Maaaaybe debug, but it's going to generate a lot of log lines

Reply:
Oh yes this was leftover from debugging